content/fr/_index.md

```diff
@@ -0,0 +1,90 @@
+---
+title: OpenTelemetry
+description: >-
+  Une télémétrie de qualité, universelle et portable pour une observabilité efficace
```

Comment:
```suggestion
  Une télémetrie de qualité, universelle et portable pour une observabilité efficace
```

In `fr-mots.txt` it is `télémetrie`

Reply:
My mistake @svrnm . I will fix the fr-mots.txt file.